include/odp/api/spec/schedule_types.h

```diff
@@ -181,6 +214,9 @@ typedef	struct odp_schedule_param_t {
 	  *
 	  * Default value is 0. */
 	uint32_t lock_count;
+
+	/** Multi event config options */
+	odp_multi_event_param_t multi_event_param;
```

Comment:
schedule_param_t is a queue create parameter. Scheduler burst size config belongs to odp_schedule_config_t, which has global scheduler configuration parameters.